receiver/prometheusreceiver/README.md

```diff
@@ -65,3 +65,12 @@ receivers:
               regex: "(request_duration_seconds.*|response_duration_seconds.*)"
               action: keep
 ```
+
+To include `job` and `instance` as regular labels, use the setting `include_resource_labels: true` (defaults to `false`). For example: 
+
+```yaml
+receivers:
+  prometheus:
+    include_resource_labels: true
```

Comment:
I'd not call "instance" and "job" resource labels which sounds like resource attributes which represents something else in Otel.

Also we should have these labels by default to make the Prometheus export work. I would add an option to exclude them.

Why not something like `ignore_scrape_labels`?

Reply:
I'd second this.  I think the implementation looks sound, but it would be good to avoid potential confusion of these labels with OTel resource attributes.  I'd also think that they should be included by default and a name like `ignore_scrape_labels` makes it clear that the user is choosing to disable something otherwise useful.

Reply:
A the time of this PR, both instance & job did not come from receiver. Now that they are, should we just not do anything here (meaning just close this PR) ?

Reply:
@rakyll @Aneurysm9 Do you have an opinion about Vishwa's above comment about how the instance and job labels are now resource attributes (at the original time of this PR they were not)? Are we good with just having them as resource attributes or do we still want them as actual labels, too, which this PR would do? If so, I can make the changes with `ignore_scrape_labels` suggested above.

Reply:
I think this PR can be closed as the `job` and `instance` labels are always included via #2897.  This PR would have the advantage of exposing configuration to disable inclusion of those labels, but they are required for Prometheus remote write compatibility and giving the user more knobs to tweak isn't always ideal.